Source/WebCore/accessibility/AccessibilityScrollView.h

```diff
@@ -44,6 +44,7 @@ class AccessibilityScrollView final : public AccessibilityObject {
     virtual ~AccessibilityScrollView();
 
     AccessibilityObject* webAreaObject() const override;
+    void setNeedsToUpdateChildren() override { m_childrenDirty = true; }
```

Comment:
Does this need to be public?

Reply:
Yes, now that we are calling it from AXObjectCache we need to make it public here